### PR TITLE
fix: fix that element has loading image #3

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#3

### CODE REVIEW

File: src/loaders/Loader.ts [hunk 1]
```diff
@@ -12,11 +12,12 @@ import { removeEvent, hasSizeAttribute, hasLoadingAttribute, addEvent } from "..
 export default abstract class Loader<T extends HTMLElement = any> extends Component<ImReadyLoaderEvents> {
   public static EVENTS: string[] = [];
   public options!: ImReadyLoaderOptions;
+  public isReady = false;
+  public isPreReady = false;
+  public hasDataSize = false;
+  public hasLoading = false;
```
Comment (mixed) [inline src/loaders/Loader.ts:18]: If you have changed it for testing, how about using [getter](https://stackoverflow.com/a/35991491)?

File: test/unit/utils.ts [hunk 2]
```diff
@@ -72,5 +72,6 @@ export function expectOrders(events: EventType[], values: string[]) {
   // Then
   const eventNames = events.map(e => e.eventType);
 
+  console.log(eventNames);
```
Comment (mixed) [inline test/unit/utils.ts:75]: remove?

File: src/loaders/ImageLoader.ts [hunk 3]
```diff
@@ -8,13 +8,9 @@ import Loader from "./Loader";
 
 export default class ImageLoader extends Loader<HTMLImageElement> {
   public static EVENTS = ["load", "error"];
-  public isPreReady() {
```
Comment (mixed) [inline src/loaders/ImageLoader.ts:11]: Has the interface changed?

**Review by mixed — APPROVED** [review]
🤩🤩🤩🤩🤩

File: src/loaders/ImageLoader.ts (hunk 3 above)
Comment (daybrush) [reply src/loaders/ImageLoader.ts:11]: @mixed 

loader is a class that can only be used inside, this method is used internally.

**Review by WoodNeck — APPROVED** [review]
LGTM

**Pull request merged**